### DISCUSSION TITLE
pimcore_WYSIWYG not working after upgrade

### DISCUSSION THREAD

**Discussion by ewolfMM** [pimcore/pimcore#17565] [Q&A / Support]
We got a problem: Wysiwyg content not being saved to database.
if  using {{ pimcore_wysiwyg("myWYSIWYG", {
            "height": 600,
            "width": 1100,
            "placeholder": "Enter you content"
        }) }} in out template, enter some content and publish, nothing is being saved to the database. after a reload the editor field remains empty. Bundle is registered. ideas?

**Comment by plixus** [comment]
I had a similar problem. If the text in a WYSIWYG field was changed and save was clicked, it was not saved when the content element was reloaded. In the object editor I had this Editor Configuration:

```
{
    “plugins": ‘lists link code table’,
    “toolbar1": ”undo redo | blocks | bold superscript | table | bullist numlist link | removeformat code”
}
```

After I removed the plugins line, it worked.
